ccmlib/scylla_node.py

```diff
@@ -1348,6 +1348,45 @@ def gnutls_config_file(self):
                 return str(candidate)
         raise ValueError(f"gnutls.config wasn't found in any path: {candidates}")
 
+    def run_scylla_sstable(self, command, additional_args=None, keyspace=None, datafiles=None, column_families=None, batch=False):
+        """Invoke scylla-sstable, with the specified command (operation) and additional_args.
+
+        For more information about scylla-sstable, see https://docs.scylladb.com/stable/operating-scylla/admin-tools/scylla-sstable.html.
+
+        Params:
+        * command - The scylla-sstable command (operation) to run.
+        * additional_args - Additional command-line arguments to pass to scylla-sstable, this should be a list of strings.
+        * keyspace - Restrict the operation to sstables of this keyspace.
+        * datafiles - Restrict the operation to the specified sstables (Data components).
+        * column_families - Restrict the operation to sstables of these column_families. Must contain exactly one column family when datafiles is used.
+        * batch - If True, all sstables will be passed in a single batch. If False, sstables will be passed one at a time.
+            Batch-mode can be only used if column_families contains a single item.
+
+        Returns: map: {sstable: (stdout, stderr)} of all invokations. When batch == True, a single entry will be present, with empty key.
+
+        Raises: subprocess.CalledProcessError if scylla-sstable returns a non-zero exit code.
+        """
+        if additional_args is None:
+            additional_args = []
+        scylla_path = common.join_bin(self.get_path(), BIN_DIR, 'scylla')
+        env = self.get_env()
+        sstables = super()._Node__gather_sstables(datafiles, keyspace, column_families)
```

Comment:
Why use super and not self ?

Reply:
It took me a while to figure out how to call the method with `__` prefix. I guess `super()` is a leftover of things I tried. Removed in v2.